models/block.go

```diff
@@ -83,7 +78,21 @@ func decodeBlockEvent(event cadence.Event) (*Block, error) {
 	}, nil
 }
 
+// blockV0 lacks the PrevRandao field, that was included later
 type blockV0 struct {
+	Block             *blockV0Fields
```

Comment:
For some strange reason, if we define the `blockV0` type as:
```go
type blockV0 struct {
	*blockV0Fields
	TransactionHashes []gethCommon.Hash
}
```
We get the following error:
```go
rlp: input string too short for common.Hash, decoding into (models.blockV0).TransactionHashes[1]
```
when decoding the block.
That's why it's best if we have a named field for `blockV0Fields`.